### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Integrated Terminal (Managing multiple terminals)_ScreenReader: Column header not announced for cell data present under column 2. 

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/vscode-docs#3603]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#CELA P1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS4.2.3;#MAS1.3.1;#Win10;#Visual Studio Code;#CELAP1_code;#Narrator;

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/docs/editor/integrated-terminal 
Windows Version: Windows10
Browser version: Microsoft Edge Version Version 84.0.488.1 (Official build) dev (64-bit)
Screen reader: narrator 

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/docs/editor/integrated-terminal
2)Integrated Terminal Page will get open.
3)Go through "Managing multiple terminals/Configuration/Terminal display settings".
4) Check if column header announcement is made for respective column data present under the column.

**Actual:**
Column header is not announced for the data that is present under column 2.

**Same issue under following heading on same page:**
Terminal key Bindings.

**Expected:**
Column header should be announced for all the cell data that is present under that particular column after announcement of the cell data.

**User Impact:**
Screen reader users will not know the data is present under which column header.

**Attachment link for Reference:**
[3603_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_Integrated Terminal (Managing multiple terminals)_ScreenReader.zip](https://github.com/microsoft/vscode-docs/files/4527126/3603_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_Integrated.Terminal.Managing.multiple.terminals._ScreenReader.zip)

**Comment by gregvanl** [comment]
Not sure how to fix this since it is just a Markdown table. The same behavior reproduces if you review the topic on GitHub https://github.com/microsoft/vscode-docs/blob/master/docs/editor/integrated-terminal.md.
The HTML is pretty simple so I think this may be a narrator issue.

![image](https://user-images.githubusercontent.com/12818376/82617394-cddca800-9b84-11ea-937d-006b0bf27baa.png)

**Comment by ManishaC1** [comment]
Verified the issue in below environment.
URL:https://code.visualstudio.com/docs/terminal/basics
There is no table content.
Hence adding below tag.
Github tag: #Closed
![Screenshot (428)](https://user-images.githubusercontent.com/105198219/216930102-5bd0e449-d3ad-4f1d-9814-f9211a831296.png)
